vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -102,11 +105,17 @@
  *            the type of the items to be inserted in the combo box
  * @author Vaadin Ltd
  */
+@Tag("vaadin-combo-box")
+@NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "23.1.0-beta1")
+@JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
+@NpmPackage(value = "@vaadin/combo-box", version = "23.1.0-beta1")
+@NpmPackage(value = "@vaadin/vaadin-combo-box", version = "23.1.0-beta1")
+@JsModule("@vaadin/combo-box/src/vaadin-combo-box.js")
+@JsModule("@vaadin/polymer-legacy-adapter/template-renderer.js")
 @JsModule("./flow-component-renderer.js")
 @JsModule("./comboBoxConnector.js")
-public class ComboBox<T> extends GeneratedVaadinComboBox<ComboBox<T>, T>
-        implements HasSize, HasValidation,
-        HasDataView<T, String, ComboBoxDataView<T>>,
+public class ComboBox<T> extends ComboBoxBase<ComboBox<T>, T, T> implements
+        HasSize, HasValidation, HasDataView<T, String, ComboBoxDataView<T>>,
```

Comment:
We will want to move some of these interfaces to the base class as well, but maybe we should wait until all corresponding implementations have moved.